var/spack/repos/builtin/packages/libsonata-report/package.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class LibsonataReport(CMakePackage):
+    """
+    `libsonatareport` provides C++ API for writing SONATA reports
+
+    See also:
+    https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md
+    """
+    homepage = "https://github.com/BlueBrain/libsonata"
+    git = "https://github.com/BlueBrain/libsonata.git"
```

Comment:
Just for my info, we have a new package based on the same repo while this is not merged in master?

Reply:
Thats it. Since it wont be merged in libsonata repo and while we find a place of it, we use a a commit from the reports branch of libsonata to keep them separated (is a different shared library anyway).